website/pages/docs/agent/config-entries/service-resolver.mdx

```diff
@@ -117,8 +134,8 @@ Name = "web"
     resolve instead of one defined as that service's DefaultSubset If empty the
     default subset is used.
 
-        If this is specified at least one of Service, Datacenter, or Namespace
-        should be configured.
+    If this is specified at least one of Service, Datacenter, or Namespace
```

Comment:
Did changing this indent change how this section is displayed? On the old website I recall needing extra indents for some reason in the markdown for things to show up normally indented.

Reply:
That was because I noticed it was in a code-looking block:
https://www.consul.io/docs/agent/config-entries/service-resolver.html#servicesubset

Didn't seem intentional.